src/snapshot/interval_strategy.rs

```diff
@@ -0,0 +1,116 @@
+use cosmwasm_std::{Order, StdResult, Storage};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+
+use crate::{Bound, KeyDeserialize, Map, Prefixer, PrimaryKey};
+
+use super::{ChangeSet, SnapshotStrategy};
+
+/// A SnapshotStrategy that takes a snapshot only if at least the specified interval has passed.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
```

clippy complains here